### PR TITLE
MGMT-1681: Use only official AWS SDK for S3 operations

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

**Comment by avishayt** [comment]
@tsorya Please review

File: pkg/s3wrapper/client.go [hunk 1]
```diff
@@ -68,16 +98,122 @@ func NewS3Session(cfg *Config) (*session.Session, error) {
...
+			}
+			return errors.Wrap(err, fmt.Sprintf("Failed to delete object %s from bucket %s", objectName, c.cfg.S3Bucket))
+		}
+	}
+
+	log.Infof("Deleted object %s from bucket %s", objectName, c.cfg.S3Bucket)
+	return nil
+}
+
+func (c *S3Client) UpdateObjectTag(ctx context.Context, objectName, key, value string) (bool, error) {
```
Comment (tsorya) [inline pkg/s3wrapper/client.go:194]: i think name UpdateObjectTag for function that returns bool is very confusing

Comment (tsorya) [reply pkg/s3wrapper/client.go:194]: i know that it was previously the same

**Pull request merged**